package/MDAnalysis/analysis/bat.py

```diff
@@ -63,7 +63,7 @@
 --------
 :class:`~MDAnalysis.analysis.dihedrals.Dihedral`
    class to calculate dihedral angles for a given set of atoms or residues
-:func:`MDAnalysis.lib.distances.calc_dihedrals()`
+:func:`MDAnalysis.calc_dihedrals()`
```

Comment:
Why is this change made here? There is no such method as `MDAnalysis.calc_dihedrals()`, the previous name was fine?

Reply:
@IAlibay i don't how it changed , i made changes only in [package/MDAnalysis/lib/distances.py](https://github.com/MDAnalysis/mdanalysis/blob/d9272baa41f8cdbff30869e3ddfaf1bec7e07a8f/package/MDAnalysis/lib/distances.py) . sorry for that i'll fix it again